bundles/org.smarthomej.binding.tuya/src/main/java/org/smarthomej/binding/tuya/internal/handler/TuyaDeviceHandler.java

```diff
@@ -174,6 +172,14 @@ private void processChannelStatus(Integer dp, Object value) {
                     && CHANNEL_TYPE_UID_SWITCH.equals(channelTypeUID)) {
                 updateState(channelId, OnOffType.from((boolean) value));
                 return;
+            } else if (value instanceof String && CHANNEL_TYPE_UID_IR_CODE.equals(channelTypeUID)) {
+                if (configuration.dp == 2) {
+                    String decoded = convertBase64Code(configuration, (String) value);
+                    logger.warn("ir code: {}", decoded);
```

Comment:
Why is this a WARN level log? Also please include at least the thing UID, otherwise it's hard to say where this comes from.